front-end-project-week/src/views/CreateNoteView.js

```diff
@@ -24,6 +24,41 @@ const H2PageTitle = styled.h2`
  ********************************************* Component *******************************************
  **************************************************************************************************/
 class CreateNoteView extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      newNote: {
+        title: props.edit ? props.note.title : '',
+        textBody: props.edit ? props.note.textBody : ''
+      },
+      submitButtonText: 'Save'
+    };
+  }
+  //========================== Methods =========================
+  clearNewNoteState = () => {
+    this.setState({
+      newNote: {
```

Comment:
Since there's nothing else on your state, you can just have `title` and `textBody` in state directly rather than nested in an object. That way you won't have to use the spread operator on line 50 in `handleInputChange`. Then in your submitHandler method, just pass  in `this.state` on line 57.